server/src/domain/storage/storage.service.ts

```diff
@@ -26,6 +26,10 @@ export class StorageService {
 
       try {
         await this.storageRepository.unlink(file);
+        const found = file.match(/(?<path>.*\/.*)\/.*\/.*\..*/);
```

Comment:
Can you help write a few words on the purpose of this RegEx as a comment above this line?

Reply:
Yeah sure, that is a good idea!

Reply:
``` 
// extracts the file's parent folder
// upload/thumbs/89/ab/89abcd.jpeg -> upload/thumbs/89
```
would you say that is understandable?

Reply:
yes this looks good, thank you